src/test/java/kitchenpos/integrationTest/MenuServiceIntegrationTest.java

```diff
@@ -0,0 +1,174 @@
+package kitchenpos.integrationTest;
+
+import kitchenpos.application.MenuService;
+import kitchenpos.domain.*;
+import kitchenpos.fixtures.MenuFixture;
+import kitchenpos.fixtures.MenuGroupFixture;
+import kitchenpos.fixtures.MenuProductFixture;
+import kitchenpos.fixtures.ProductFixture;
+import kitchenpos.infra.PurgomalumClient;
+import kitchenpos.repository.InMemoryMenuGroupRepository;
+import kitchenpos.repository.InMemoryMenuRepository;
+import kitchenpos.repository.InMemoryProductRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.math.BigDecimal;
+import java.util.Collections;
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.BDDMockito.given;
+
+
+@ExtendWith(MockitoExtension.class)
+public class MenuServiceIntegrationTest {
+    private MenuRepository menuRepository;
+    private MenuService menuService;
+
+    private MenuGroupRepository menuGroupRepository;
+    private ProductRepository productRepository;
+
+    @Mock
+    private PurgomalumClient purgomalumClient;
+
+    @BeforeEach
+    void setUp() {
+        menuRepository = new InMemoryMenuRepository();
+        menuGroupRepository = new InMemoryMenuGroupRepository();
+        productRepository = new InMemoryProductRepository();
+        menuService = new MenuService(menuRepository, menuGroupRepository, productRepository, purgomalumClient);
+    }
+
+    @Test
+    void 메뉴를_등록_할_수_있다() {
+        // Prepare data
+        MenuGroup menuGroup = MenuGroupFixture.create("치킨메뉴");
+        menuGroupRepository.save(menuGroup);
+
+        Product product = ProductFixture.create("후라이드치킨", BigDecimal.valueOf(16000L));
```

Comment:
Fixture Method를 통해 테스트 객체를 생성하는것도 좋지만 자주 사용되거나 의미를 가지는 테스트 객체는 Fixture 클래스가 객체로 전달 해주는것도 테스트 코드의 가독성이 올라갈수 있습니다 😄 
```suggestion
        Product 후라이드치킨_16000원 = ProductFixture.후라이드치킨_16000원();
```

Reply:
현재 TestFixture를 단순 팩토리 역할로만 사용하고 계신것 같습니다. 
남겨드린 코멘트처럼 자주 사용되거나, 의미를 가지는 테스트 객체를 Fixture 클래스에 정의하고, 테스트 코드에서는 정의된 Fixture를 가져다 사용하여 가독성을 높이고 테스트 코드 작성 리소스를 줄일수 있는 경험을 해보세요 😄